plugins/org.eclipse.n4js.utils/src/org/eclipse/n4js/utils/nodemodel/NodeModelUtilsN4.xtend

```diff
@@ -16,12 +16,31 @@ import org.eclipse.xtext.nodemodel.INode
 import org.eclipse.xtext.nodemodel.util.NodeModelUtils
 import org.eclipse.xtext.util.ITextRegion
 import org.eclipse.xtext.util.TextRegion
+import org.eclipse.xtext.nodemodel.BidiIterable
+import org.eclipse.emf.ecore.EObject
 
 /**
  * Utility methods for dealing with the parse tree, in addition to those in {@link NodeModelUtils}.
  */
 class NodeModelUtilsN4 {
 
+	def public static INode findKeywordNode(ICompositeNode parentNode, String keyword) {
+		val BidiIterable<INode> iterable = parentNode.children;
```

Comment:
maybe "children" instead of "iterable" would be a better name? As a matter of fact "siblings" would be even better suited, since the children of the parent are the siblings, aren't they?